metric/reader/reader.go

```diff
@@ -24,17 +24,19 @@ import (
 	"go.opencensus.io/metric/metricdata"
 	"go.opencensus.io/metric/producer"
 	"go.opencensus.io/trace"
+	"sync"
 )
 
 // Reader periodically reads metrics from all producers registered
 // with producer manager and exports those metrics using provided
-// exporter. Call Reader.Stop() to stop the reader.
+// exporter1. Call Reader.Stop() to stop the reader1.
```

Comment:
What are "exporter1" and "reader1"? Typo?

Reply:
when refactored it renamed in reader.go as well. I'll fix it.